src/java/picard/sam/markduplicates/util/OpticalDuplicateFinder.java

```diff
@@ -87,20 +86,41 @@ public OpticalDuplicateFinder(final String readNameRegex, final int opticalDupli
 
         final PhysicalLocation actualKeeper = keeperOrNull(list, keeper);
 
+        final Log log = Log.getInstance(OpticalDuplicateFinder.class);
+        ProgressLogger progressLogger = new ProgressLogger(log, 10000, "examined", "ReadEnds to keeper");
+        if(length > 1000) {
```

Comment:
Can this be a parameter to this method?  And then have another version sets this to `1000`.  I definitely see this being adjusted by folks operating on duplicate sets.